rmgpy/data/statmech.py

```diff
@@ -400,8 +400,9 @@ def getStatmechData(self, molecule, thermoModel):
                 freqs_removed = 0
                 freq_count = len(frequencies)
                 while freq_count > num_vibrations:
-                    min_degeneracy, min_entry = min([(entry.data.symmetry, entry)
-                                                     for entry in group_count if group_count[entry] > 0])
+                    min_entry = min((entry for entry in group_count if group_count[entry] > 0),
```

Comment:
Would it be better to implement the comparison methods for the class or is it more likely that the sorting keys will be case-by-case dependent

Reply:
It seemed like Entries would generally be sorted for a specific purpose, which would require a custom sorting key. Species/Molecules/Atoms/Bonds were often sorted for deterministic-ness, so it seemed appropriate to implement comparison methods.